tools/clang_tidy.py

```diff
@@ -8,28 +8,35 @@
 import sys
 
 
-DEFAULT_FILE_PATTERN = r".*\.[ch](pp)?"
+DEFAULT_FILE_PATTERN = r".*\.c(c|pp)?"
 
 # @@ -start,count +start,count @@
 CHUNK_PATTERN = r"^@@\s+-\d+,\d+\s+\+(\d+)(?:,(\d+))?\s+@@"
 
 
-def run_shell_command(arguments, process_name=None):
+def run_shell_command(arguments, process_name=None, verbose=False):
     """Executes a shell command."""
     assert len(arguments) > 0
     try:
-        output = subprocess.check_output(arguments, stderr=subprocess.STDOUT)
```

Comment:
I don't see why you're piping stderr to stdout. Doesn't seem legit.

Reply:
otherwise the output is emitted to the console which is noisy. This way it's all aggregated in one stream that is ignored or printed to the user in verbose mode